src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ToGerritRunListener.java

```diff
@@ -81,10 +86,10 @@ public static ToGerritRunListener getInstance() {
     }
 
     @Override
-    public synchronized void onCompleted(AbstractBuild r, TaskListener listener) {
+    public synchronized void onCompleted(AbstractBuild r, @Nonnull TaskListener listener) {
         GerritCause cause = getCause(r);
         logger.debug("Completed. Build: {} Cause: {}", r, cause);
-        if (cause != null) {
+        if (r != null && cause != null) {
```

Comment:
But in fact it should be `@Nonnull Run`. Can fix that later in core, but in the meantime I think you can just add this annotation to your override.